tests/components/systemmonitor/conftest.py

```diff
@@ -2,10 +2,35 @@
 from __future__ import annotations
 
 from collections.abc import Generator
-from unittest.mock import AsyncMock, patch
+import socket
+from unittest.mock import AsyncMock, Mock, patch
 
+from psutil import NoSuchProcess, Process
+from psutil._common import sdiskpart, sdiskusage, shwtemp, snetio, snicaddr, sswap
+from psutil._pslinux import svmem
```

Comment:
This fails to import when testing on macos

Reply:
Maybe something like
```

@dataclass
class MockSvMem:

    total: int
    available: int
    percent: float

```

Reply:
FAILED tests/components/systemmonitor/test_init.py::test_load_unload_entry - AttributeError: Mock object has no attribute 'sensors_temperatures'

`sensors_temperatures` also doesn't exist on macos

Reply:
> `sensors_temperatures` also doesn't exist on macos

According to `psutil` it should https://github.com/giampaolo/psutil/blob/master/psutil/__init__.py#L2284

Reply:
```
ImportError while loading conftest '/Users/bdraco/home-assistant/tests/components/systemmonitor/conftest.py'.
tests/components/systemmonitor/conftest.py:10: in <module>
    from psutil._pslinux import svmem
venv/lib/python3.12/site-packages/psutil/_pslinux.py:25: in <module>
    from . import _psutil_linux as cext
E   ImportError: cannot import name '_psutil_linux' from 'psutil' (/Users/bdraco/home-assistant/venv/lib/python3.12/site-packages/psutil/__init__.py). Did you mean: '_psutil_osx'?

```

Reply:
Not sure if this helps
```python
@pytest.fixture(autouse=True)
def mock_sys_platform() -> Generator[None, None, None]:
    """Mock sys platform to Linux."""
    with patch("sys.platform", "linux"):
        yield
```